distributed-map/criteria-api/src/main/java/PredicateMember.java

```diff
@@ -9,10 +9,7 @@
 import java.util.HashSet;
 import java.util.Set;
 
-import static com.hazelcast.query.Predicates.and;
-import static com.hazelcast.query.Predicates.equal;
-import static com.hazelcast.query.Predicates.not;
-import static com.hazelcast.query.Predicates.or;
+import static com.hazelcast.query.Predicates.*;
```

Comment:
We don't use star imports in Hazelcast code, actually CheckStyle should fail on this.

![screenshot from 2017-08-08 17-18-47](https://user-images.githubusercontent.com/4196298/29079524-b6005eaa-7c5d-11e7-816e-f5ddc29b214f.png)
That's one of the few things you have to change in the IntelliJ code formatter to make it Hazelcast compatible.